pandas/core/reshape/reshape.py

```diff
@@ -697,7 +697,7 @@ def _convert_level_number(level_num, columns):
 
 
 def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
-                columns=None, sparse=False, drop_first=False):
+                columns=None, sparse=False, drop_first=False, dtype=None):
```

Comment:
Any reason not to use `'uint8'` or `np.uint8` here?

Reply:
I've tried to mirror API of DataFrame, Series, Panel etc. where passing None explicitly is allowed and means "dtype will be inferred".

Reply:
@jreback @TomAugspurger So this is the last question to answer. Do you accept my argument about None or should I change it to np.uint8?

Reply:
this is ok here, it follows a similar style elsewhere